modules/op-matching-pipeline-run-using-pipelines-as-code.adoc

```diff
@@ -0,0 +1,208 @@
+[id="matching-pipeline-run-using-pipelines-as-code_{context}"]
+= Matching an event to a pipeline run
+
+You can match different Git provider events with each pipeline run by using special annotations on the pipeline run. If there are multiple pipeline runs matching an event, {pac} runs them in parallel and posts the results to the Git provider as soon a pipeline run finishes.
+
+[discrete]
+.Matching a pull event to a pipeline run
```

Comment:
These still look like they should be discrete modules to me. Even if not, the titles should be headings rather than labels.

Reply:
I would've also expected these to be separate modules. Although they aren't labeled as procedures, technically they kind of are, because to achieve the matching, the user is annotating a YAML file with what is in the provided example. (I think?) I see that they are now organized by separate headings, but the idea of modularization is to avoid the really long modules with different headings. I understand that it's more common with reference modules, but I will also cast my vote for splitting these (sorry!).